packages/common/src/converter/payload-converters.ts

```diff
@@ -87,3 +141,27 @@ export class BinaryPayloadConverter implements PayloadConverterWithEncoding {
     return content.data as any;
   }
 }
+
+export const searchAttributePayloadConverter = new WrappedPayloadConverter(new JsonPayloadConverter());
+
+export class DefaultPayloadConverter extends CompositePayloadConverter {
+  // Match the order used in other SDKs, but exclude Protobuf converters so that the code, including
+  // `proto3-json-serializer`, doesn't take space in Workflow bundles that don't use Protobufs. To use Protobufs, use
+  // {@link DefaultPayloadConverterWithProtobufs}.
+  //
+  // Go SDK:
+  // https://github.com/temporalio/sdk-go/blob/5e5645f0c550dcf717c095ae32c76a7087d2e985/converter/default_data_converter.go#L28
```

Comment:
Do we need this link?

Reply:
I like it as a reference to what the order should be. Until we have a spec 😄